lib/python/gunittest/multirunner.py

```diff
@@ -147,7 +153,9 @@ def main():
                               '-m', 'grass.gunittest.main',
                               '--grassdata', gisdb, '--location', location,
                               '--location-type', location_type,
-                              '--output', absreport],
+                              '--output', absreport,
+                              '--test_list_in', test_list_in,
+                              '--test_list_out', test_list_out],
```

Comment:
multirunner and output file does not make sense. If nothing else, the current implementation just overwrites the file.